### DISCUSSION TITLE
🎉 New Feature: Auto-Hide on desktop (v1.7.27)

### DISCUSSION THREAD

**Discussion by chrisgurney** [chrisgurney/obsidian-note-toolbar#71] [Announcements]
Version 1.7.27 adds the ability to auto-hide toolbars using a new `auto-hide` style. When applied, the toolbar is hidden from view until the area it's in is hovered over. (Note that it only applies on desktop.)

![CleanShot 2024-05-14 at 21 21 40](https://github.com/chrisgurney/obsidian-note-toolbar/assets/17337703/c19769c5-5055-4143-bad0-f5de4e8e692c)

Thanks again to @fredjoe7 for the suggestion!

**Comment by fredjoe7** [comment]
Thank you.
